infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -390,7 +408,25 @@ public void resetServer(CorfuPayloadMsg<SequencerTailsRecoveryMsg> msg,
 
             // Clear the existing map as it could have been populated by an earlier reset.
             streamTailToGlobalTailMap.clear();
-            streamTailToGlobalTailMap.putAll(streamTails);
+
+            // Set tail for every stream
+            // TODO (Anny): assess cost of computation vs. sending stream tails as well on reset
```

Comment:
Please get rid of todos, they are considered as a bad practice http://wiki.c2.com/?TodoCommentsConsideredHarmful